MM-30516. Updated Add Service task.

rachelbritton opened the pull request:
Updated the instructions for Adding a Service, including new images.

rachelbritton committed:
MM-30516. Updated Add Service task.

Updated the instructions for Adding a Service, including new images.

ethorlacius commented on guides/admin-manual/webapp_server_management.rst:
```diff
@@ -104,36 +104,37 @@ Micetro requires an agent to handle communication with external services. The in
+      :width: 100%
 
-3. Select an agent: 
+3. On the :guilabel:`Properties` tab, select an agent from the dropdown menu.
```
I'd use the word "step" instead of "tab" for the pages in a wizard. There is a subtle difference, tabs are shown at the top of the form and the user can switch back and forth, whereas the steps are listed down the left side and it is assumed that the user goes through them in order

ethorlacius commented on guides/admin-manual/webapp_server_management.rst:
```diff
@@ -104,36 +104,37 @@ Micetro requires an agent to handle communication with external services. The in
+   * With the exception of the above services, the agent can be installed on the Micetro Central host, the service host, or --- for MS DNS/DHCP, ISC Kea, Cisco Meraki, and Cisco IOS --- a different host. If you have already added a service of this type, you can select from existing agents or create a new one. A single agent can be used to manage multiple service connections.
 
-   * In the case of **AuthServe**, you can select from a list of available agents that were registered in the installation process. You can also register a new agent on the :guilabel:`New Agent` tab. Enter a hostname for the agent and, optionally, the IP address. If the hostname is not resolvable, an IP address is needed here.
+   * In the case of **AuthServe**, you can select from a list of available agents that were registered in the installation process. You can also register a new agent by selecting :guilabel:`New Agent` from the :guilabel:`Agent` dropdown. Enter a hostname or IP address (if the agent hostname is not resolvable), as well as the agent secret.
```
The user can select agents when adding AuthServe, Cisco IOS, Kea, Meraki, MS DNS and MS DHCP. There are some differences between the service types: for AuthServe the user must specify the name and shared secret for the agent, for all but AuthServe, Meraki and Cisco he can select "Service host" to use the agent installed on the host he is adding

On a related note, Alexander was just pointing out to me that in the new wizard it is only possible to select "New agent" for AuthServe but in the old one it could be done for all the other service types as well. I'm hoping I'll be able to squeeze a fix for it into 11.2. If I do, I'll let you know

rachelbritton replied:
I will be updating the screenshots here because they show the UI before the text was adjusted. Then, I'll re-request your review.

rachelbritton committed:
MM-30516. Changed tab to step.

Changed all instances of "tab" to "step".

rachelbritton committed:
MM-30516. Updated images and added note.

Updated images post UI text revisions and added note about agents/service host.
